static/lang/re-en.json

```diff
@@ -1572,6 +1572,9 @@
             "Fighter": {
                 "DoubleSlice": {
                     "SecondAttackLabel": "Double Slice Second Attack"
+                },
+                "SnaggingStrike": {
+                    "Note": "If this Strike hits, the target is @UUID[Compendium.pf2e.conditionitems.AJh5ex99aV6VTggg]{Flat-Footed} until the start of your next turn or until it's no longer within the reach of your hand, whichever comes first."
```

Comment:
I wonder if we actually need this one? It's all the feat does.

Reply:
Yeah, that’s true. Was mostly a way to deliver a convenient drag link for Flat-Footed. I’m happy to close this if you don’t think it’s needed.

Reply:
I think they can just post the ability itself.